voluptuous/schema_builder.py

```diff
@@ -1127,8 +1127,10 @@ class Inclusive(Optional):
     True
     """
 
-    def __init__(self, schema, group_of_inclusion, msg=None, description=None):
+    def __init__(self, schema, group_of_inclusion,
+                 msg=None, default=UNDEFINED, description=None):
```

Comment:
Add new parameters at the end to avoid unnecessary breakage.

Reply:
OK. I put them in that order because that is the order used by both `Required` and `Optional` (which is its parent class) but I'm happy either way.

Reply:
I've pushed https://github.com/alecthomas/voluptuous/pull/382/commits/d6e83a6b4aa29e1c18361a0057b5fdae9b00f251 to address this. Reordering the arguments is the only thing that commit does, so If you decide that consistency with the other classes is more compelling than consistency with the part then feel fee to ignore it.

Reply:
Ah understood, but still I think avoiding potential breakage is preferable.